src/components/proofs/EmailProofForm.tsx

```diff
@@ -43,17 +47,17 @@ export default function ({
     onChange('')
   }
 
-  async function onSendEmail(email: string) {
+  async function onSendEmails(emails: readonly string[]) {
```

Comment:
Should we move to the helper this function? With `onGenerateProof` it looks messy

Reply:
I can, but I'll need to pass at least 3 additional callback function into it, I think it will be harder to see what's going on